packages/events/src/service/indexing/indexing.ts

```diff
@@ -286,8 +287,32 @@ export async function getIndexedEvents() {
     return []
   }
 
+  const query = {
+    // We basically want to fetch all events,
+    // UNLESS they are in status 'CREATED' (i.e. undeclared drafts) and not created by current user.
+    bool: {
+      should: [
+        {
+          bool: {
+            must_not: [{ term: { status: EventStatus.CREATED } }]
+          }
+        },
+        {
+          bool: {
+            must: [
+              { term: { status: EventStatus.CREATED } },
+              { term: { createdBy: userId } }
+            ]
+          }
+        }
+      ],
+      minimum_should_match: 1
+    }
+  } as estypes.QueryDslQueryContainer
```

Comment:
Unfortunately we need the casting due to issue with Chai types:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/43744

This could be fixed, but might require updating vitest to newest version on both events and client, which is a larger task.